tao/CMake/Googletest/CMakeLists.txt.in

```diff
@@ -4,14 +4,13 @@ project(googletest-download NONE)
 
 include(ExternalProject)
 ExternalProject_Add(googletest
-  GIT_REPOSITORY    https://github.com/google/googletest.git
-  GIT_TAG           main 
-  GIT_SHALLOW       1
-  SOURCE_DIR        "${CMAKE_BINARY_DIR}/googletest-src"
-  BINARY_DIR        "${CMAKE_BINARY_DIR}/googletest-build"
-  CONFIGURE_COMMAND ""
-  BUILD_COMMAND     ""
-  INSTALL_COMMAND   ""
-  TEST_COMMAND      ""
+  URL                   "https://pai-blade.oss-cn-zhangjiakou.aliyuncs.com/gtest/release-1.11.0.tar.gz"
```

Comment:
Nit: I think it is better to make the ExternalProject's URL configurable and only access to internal URLs inside Alibaba.

Reply:
Agree with that, will update this PR ASAP.